components/brave_rewards/resources/rewards_panel/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html dir="$i18n{textdirection}" lang="$i18n{language}">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Rewards Panel</title>
+  <link rel="stylesheet" href="chrome://brave-resources/css/reset.css">
+  <link rel="stylesheet" href="chrome://brave-resources/fonts/poppins.css">
+  <link rel="stylesheet" href="chrome://brave-resources/fonts/muli.css">
+  <script src="chrome://resources/js/load_time_data.js"></script>
+  <script src="/strings.js"></script>
+  <script type="module" src="/brave_rewards_panel.bundle.js"></script>
+</head>
```

Comment:
Perhaps try having inline css with a background color consistent with the to-be-rendered react background color, dependent on dark / light theme. That may avoid the flashes especially when we're not caching the webcontents.

Reply:
Good call!